CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+([Français](#comment-contribuer))
+
+## How to Contribute
+
+When contributing, post comments and discuss changes you wish to make via Issues.
```

Comment:
Are we going to maintain a open community or only allowing that discussion to be initiated via the Principal Publisher service desk?

Please adjust as per our intention on how we will receive and be notified of the intention to contribute.

Reply:
I removed this portion, because as discussed, Issues section was turned off.